src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+let populationsList = document.getElementsByClassName('population');
```

Comment:
why `let`? don't change value of variables if it's unnecessary
```suggestion
const populationsList = document.getElementsByClassName('population');
```

Reply:
fixed